packages/governance/src/types.js

```diff
@@ -445,6 +463,16 @@
  * @property {CreateQuestion} createQuestion
  */
 
+/**
+ * @typedef {object} GovernorFacet
+ * @property {() => ParamManagerRetriever} getParamManagerRetriever
```

Comment:
No code implements this property.
```sh
❯ git grep getParamManagerRetriever
packages/governance/docs/contractGovernance.puml:        -getParamManagerRetriever()
packages/governance/src/types.js: * @property {() => ParamManagerRetriever} getParamManagerRetriever
```

I think you want `getParamMgrRetriever`. This would have been caught if the new type were used to annotate on object, so that might be worth doing. E.g. at https://github.com/Agoric/agoric-sdk/blob/d6bba43627c64c6a130e5fcb12bda091ae51522b/packages/governance/src/contractHelper.js#L132 

When I add this annotation,
```
    /** @type {import('@agoric/vat-data/src/types.js').FunctionsPlusContext<unknown, GovernorFacet>} */
```
I get a type error for `setOfferFilter`, which I think is correct because it's a virtual object so its first param must be the context (like `getGovernedApiNames` above it):
https://github.com/Agoric/agoric-sdk/blob/d6bba43627c64c6a130e5fcb12bda091ae51522b/packages/governance/src/contractHelper.js#L144-L146

This makes me think that `makeVirtualGovernorFacet` has a bug that no runtime tests are detected. So I'm going to request changes that the type be annotated here and that there be a runtime test that confirms that zcf got valid strings here.

Reply:
I discovered the same thing in the new test of offerFilter for PSM (#6047). It resulted in changing the parameters to `getInvitation` in `makeGovernorFacet` as well.

I added the annotation and also added it for `makeGovernorFacet`.